shared/wallets/link-existing/index.stories.tsx

```diff
@@ -25,7 +24,7 @@ const common = {
 
 const enterKeyProps = {
   ...common,
-  view: 'key',
+  view: 'key' as 'key',
```

Comment:
better way to do this?  type View = 'key' | 'name'

Reply:
You can do this:
`'key' as const` but our eslint isn't happy with it for some reason.

also see this: 
https://github.com/typescript-eslint/typescript-eslint/issues/312

Reply:
https://github.com/typescript-eslint/typescript-eslint/pull/390

We might have to update our eslint